example/CMakeLists.txt

```diff
@@ -1,7 +1,7 @@
-add_custom_target(examples DEPENDS examples)
+add_custom_target(examples DEPENDS examplesae)
```

Comment:
Right, but how it works since this `CMakeLists.txt` is inside the `example/` directory itself. 

The docs say nothing about directories, just single files. How it actually works remains unclear to me. It doesn't harm though.

https://cmake.org/cmake/help/latest/command/add_custom_target.html
> `DEPENDS`
> 
>     Reference files and outputs of custom commands created with add_custom_command()
>     command calls in the same directory (CMakeLists.txt file).